src/views/HomeView.vue

```diff
@@ -9,7 +9,19 @@
       </div>
 
       <div class="textarea-block">
-        <TextArea :value="outputTextarea" />
+        <TextArea :value="outputTextarea" :readonly="true" />
+        <Button
+          v-clipboard:copy="outputTextarea"
+          v-clipboard:success="copySuccess"
+          v-clipboard:error="copyError"
+          :class="copyBtnClass"
+          @click.once="isCopyMessage = !isCopyMessage"
+        >
+          <IconSvg :icon="icon" />
+        </Button>
+        <transition name="fade">
+          <Tooltip v-if="isCopyMessage" :message="copyMessage" />
```

Comment:
```suggestion
          <Tooltip v-if="copyMessage" :message="copyMessage" />
```


Just a suggestion. You can make this. If `copyMessage == ''`, it's false
And you can get rid of `isCopyMessage`
But your code is absolutely ok.
Seems you use `isCopyMessage` to manipulate copying? See the comment above